src/pages/edge/adobe-journey-optimizer-decisioning/optimize-configuration-view.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Optimize Setup Validation view
+description: An overview of how the Optimize datastream and sdk setup can be validated.
+keywords:
+- Datastream Validation
+- SDK Validation
+---
+
+# Optimize Setup Validation view
+
+## Overview
+
+You can use the Optimize Setup Validation view to check if your application has been set up successfully to use Adobe Experience Platform Optimize. The view validates all pre-requisites have been met and offers feedback on how to fix your setup in cases where requirements have been missed.
+
+## Getting started
+
+To use this view, follow the steps outlined below:
+
+1. [Setup](./assurance-setup.md) an Assurance session
+2. In the **Assurance Session home** view, select **Validate Setup**
+
+![Validate Setup](./assets/optimize-configuration-view/optimize-setup-view.png)
+
+## Datastream setup validation
+
+At the top of the Optimize Setup Validation view, Assurance inspects your current application's datastream to see if Target or Offer Decisioning has been enabled. In order to use optimize, at least one of extensions needs to be enabled inside your datastream.
```

Comment:
Wherever you have Optimize Setup Validation view, please change it to Decisioning Validation view

Reply:
optimize -> Optimize SDK